### ISSUE TITLE
Bring Editor toolbar & keyboard shortcuts to Comment area

### ISSUE THREAD

**Issue by amywtlin** [forem/forem#16781]
In conjunction with #16780, let's add the toolbar & keyboard shortcut experience to the comment area!

The comment area doesn't have a markdown toolbar as we have for the post editor. We all know how awesome our markdown toolbar is in the post editor, so why not in the comment area as well 🤷🏻.

Things to keep in mind,
1. Reposition **Templates** inside the overflow menu
2. Same interaction as post editor - Horizontal scroll for smaller devices and overflow menu on the right for larger devices.
3. All other interactions/keyboard shortcuts will stay as it is.

![Image](https://user-images.githubusercontent.com/84626054/174943087-ea0e758a-f5e1-42ca-b2ac-54b5e53ccf11.png)

**Comment by ludwiczakpawel** [comment]
I'll put it on my plate as I think this requires some design input first.

**Comment by amywtlin** [comment]
@ludwiczakpawel moving this back to backlog right now - please move it back to `to do - small-ish` when we are ready!

**Comment by amywtlin** [comment]
With closing #16780, we have added Embed icon to our editor toolbar which now unblocks this issue and all we need is to get the design ready to move this forward.

**Comment by aitchiss** [comment]
@amywtlin can I check if the markdown toolbar should also be shown for comments on the `/notifications` page? I (somewhat unintentionally) added it in https://github.com/forem/forem/pull/18062, but there were a few issues on this page, and I have a ["quick fix" to remove it](https://github.com/forem/forem/pull/18112) from that area for now (pending this discussion 😄).

On the notifications page, we show a much reduced version of the comment area. This is how it used to look (and will look again after my "quick fix" is merged):

![notifications page comment area](https://user-images.githubusercontent.com/20773163/178933793-845f9c26-adff-4036-9bac-c90c4a605ffc.png)

On this page we are not currently showing:
- Image upload
- Templates

**My question**:

Do we want to introduce the toolbar to comment areas on this page? If yes, do we also want to introduce template responses to this page (or do we hide this button in the toolbar)?

**Comment by aitchiss** [comment]
OK great! I'll create a separate issue for it, and we can introduce it as a follow-on PR, with all the templates functionality baked in too ✨

**Comment by amywtlin** [comment]
Sounds great! Thanks so much, @aitchiss 🥰

**Comment by aitchiss** [comment]
https://github.com/forem/forem/issues/18114
